protocol-units/execution/maptos/framework/releases/README.md

```diff
@@ -13,4 +13,217 @@ We use this directory to track framework releases and migrations. The path of mi
+| 21 | DELEGATION_POOL_PARTIAL_GOVERNANCE_VOTING  | DISABLE | ENABLE  |                  |
+| 40 | VM_BINARY_FORMAT_V7                        | DISABLE | ENABLE  | DISABLE          |
+| 46 | KEYLESS_ACCOUNTS                           | ENABLE  | ENABLE  | DISABLE          |
+| 47 | KEYLESS_BUT_ZKLESS_ACCOUNTS                | ENABLE  | DISABLE | DISABLE          |
+| 48 | REMOVE_DETAILED_ERROR_FROM_HASH            | ENABLE  | DISABLE | DISABLE          |
+| 54 | KEYLESS_ACCOUNTS_WITH_PASSKEYS             | ENABLE  | DISABLE | DISABLE          |
+| 64 | NEW_ACCOUNTS_DEFAULT_TO_FA_APT_STORE       | DISABLE | ENABLE  |                  |
+| 67 | CONCURRENT_FUNGIBLE_BALANCE                | ENABLE  | DISABLE |                  |
+| 71 | DISALLOW_USER_NATIVES                      | DISABLE | ENABLE  |                  |
+| 72 | ALLOW_SERIALIZED_SCRIPT_ARGS               | DISABLE | ENABLE  |                  |
+| 74 | ENABLE_ENUM_TYPES                          | DISABLE | ENABLE  |                  |
+| 76 | REJECT_UNSTABLE_BYTECODE_FOR_SCRIPT        | DISABLE | ENABLE  |                  |
+| 77 | FEDERATED_KEYLESS                          | DISABLE | ENABLE  |                  |
+| 78 | TRANSACTION_SIMULATION_ENHANCEMENT         | DISABLE | ENABLE  |                  |
+| 79 | COLLECTION_OWNER                           | DISABLE | ENABLE  |                  |
+| 80 | NATIVE_MEMORY_OPERATIONS                   | DISABLE | ENABLE  |                  |
+| 81 | ENABLE_LOADER_V2                           | DISABLE | ENABLE  |                  |
+| 82 | DISALLOW_INIT_MODULE_TO_PUBLISH_MODULES    | DISABLE | ENABLE  |                  |
+| 90 | NEW_ACCOUNTS_DEFAULT_TO_FA_STORE           | DISABLE | ENABLE  |                  |
+| 91 | DEFAULT_ACCOUNT_RESOURCE                   | DISABLE | ENABLE  |                  |
+| XX | GOVERNED_GAS_POOL                          | NONE    | ENABLE  | REMOVED          |
+
+## Feature description
+### 4 APTOS_STD_CHAIN_ID_NATIVES
+Activate this function `native public fun get(): u8;` which allow to access the chain Id inside Move code.
+
+Enable in Movement network and disable in Aptos. Value after migration ? 
+
+### 6 PERIODICAL_REWARD_RATE_DECREASE
+Enables scheduled reductions in validator/staker reward rates over epochs.
+
+Disable in Movement network and enable in Aptos. Value after migration ? 
+### 17 PARTIAL_GOVERNANCE_VOTING
+Changes how governance proposals are resolved when not all validators vote
+
+When this feature is disabled (default):
+ - A proposal only passes if quorum is reached and a majority of the total voting power approves.
+ - If not enough validators vote, the proposal fails by default.
+
+When this feature is enabled:
+ - The proposal outcome only considers the validators who actually voted.
+ - Abstention is no longer equivalent to a "no" vote.
+
+Disable in Movement network and enable in Aptos. Value after migration ? 
```

Comment:
I experimented with this functionality in governance testing. Because we will have a very limited validator set after the migration. We should ENABLE this feature.